docs/contribution.md

```
@@ -28,27 +28,18 @@ $ cd anta
+!!! tip "Verify editable installation"
+    Run `pip list -e` from the repository root to confirm the editable install:
 
-     ```
-      $ cd anta/anta
-      # Verify installation
-      $ pip list -e
-      Package Version Editable project location
-      ------- ------- --------------------------
-      anta    1.8.0   /mnt/lab/projects/anta
-     ```
+    ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)